src/test/java/org/fisco/bcos/channel/test/protocol/EnumNodeVersionTest.java

```diff
@@ -0,0 +1,135 @@
+                            EnumNodeVersion.BCOS_2_0_0.getVersion()),
+                    is(false));
+            assertThat(
+                    EnumNodeVersion.channelProtocolHandleShakeSupport(
+                            EnumNodeVersion.BCOS_2_0_1.getVersion()),
+                    is(false));
+            assertThat(
+                    EnumNodeVersion.channelProtocolHandleShakeSupport(
+                            EnumNodeVersion.BCOS_2_1_0.getVersion()),
+                    is(true));
+        } catch (ChannelPrococolExceiption e) {
+            assertThat(true, is(false));
+        }
+    }
+
+    @Test
+    public void EnumNodeVersionTestFunc1() {
+        try {
+            assertThat(EnumNodeVersion.channelProtocolHandleShakeSupport("1.1.0"), is(false));
+            assertThat(EnumNodeVersion.channelProtocolHandleShakeSupport("1.2.3"), is(false));
+
+            assertThat(EnumNodeVersion.channelProtocolHandleShakeSupport("2.0.0-rc1"), is(false));
+            assertThat(EnumNodeVersion.channelProtocolHandleShakeSupport("2.0.0-rc2"), is(false));
+            assertThat(EnumNodeVersion.channelProtocolHandleShakeSupport("2.0.0-rc3"), is(false));
+            assertThat(EnumNodeVersion.channelProtocolHandleShakeSupport("2.0.0-rc4"), is(false));
+            assertThat(EnumNodeVersion.channelProtocolHandleShakeSupport("2.0.0-rc5"), is(false));
+
+            assertThat(EnumNodeVersion.channelProtocolHandleShakeSupport("2.0.0"), is(false));
+            assertThat(EnumNodeVersion.channelProtocolHandleShakeSupport("2.0.1"), is(false));
+            assertThat(EnumNodeVersion.channelProtocolHandleShakeSupport("2.0.2"), is(false));
+            assertThat(EnumNodeVersion.channelProtocolHandleShakeSupport("2.0.3"), is(false));
+            assertThat(EnumNodeVersion.channelProtocolHandleShakeSupport("2.0.4"), is(false));
+
+            assertThat(EnumNodeVersion.channelProtocolHandleShakeSupport("2.1.0"), is(true));
+            assertThat(EnumNodeVersion.channelProtocolHandleShakeSupport("2.1.1"), is(true));
+            assertThat(EnumNodeVersion.channelProtocolHandleShakeSupport("2.1.2"), is(true));
+            assertThat(EnumNodeVersion.channelProtocolHandleShakeSupport("2.1.12"), is(true));
+
+            assertThat(EnumNodeVersion.channelProtocolHandleShakeSupport("2.2.0."), is(true));
+            assertThat(EnumNodeVersion.channelProtocolHandleShakeSupport("2.2.0."), is(true));
+            assertThat(EnumNodeVersion.channelProtocolHandleShakeSupport("2.2.0"), is(true));
+            assertThat(EnumNodeVersion.channelProtocolHandleShakeSupport("2.3.1-aaa"), is(true));
+            assertThat(EnumNodeVersion.channelProtocolHandleShakeSupport("2.4.2"), is(true));
+            assertThat(EnumNodeVersion.channelProtocolHandleShakeSupport("2.4.3-rc1"), is(true));
+
+        } catch (ChannelPrococolExceiption e) {
+            assertThat(true, is(false));
+        }
+    }
+
+    @Test
+    public void EnumNodeVersionTestFunc2() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The JUnit 4 test method name 'EnumNodeVersionTestFunc2' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/fisco/web3sdk/pullRequest?prid=4840376)